src/DotNetLightning.Core/Utils/Primitives.fs

```diff
@@ -338,12 +338,18 @@ module Primitives =
         member this.AsNBitcoinFeeRate() =
             this.Value |> uint64 |> (*)4UL |> Money.Satoshis |> FeeRate
 
+        member this.MismatchRatio (other: FeeRatePerKw) =
```

Comment:
Would you mind adding a doc-comment with `///` for this method that the returned value is agnostic to the absolute feerate and it is always positive? It may be good to add a reference to the wiki (https://en.wikipedia.org/wiki/Relative_change_and_difference) too.